main.py

```diff
@@ -1,10 +1,41 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players_data:
+        players = json.load(players_data)
+
+    for player in players.keys():
```

Comment:
you can use `for player, player_info in player.items()`, it allows you to use `player_info["email"] instead of players[player]["email"]`